exampleSite/content/fragments/404/code-404.md

```diff
@@ -0,0 +1,25 @@
++++
+fragment = "content"
+weight = 120
++++
+
+<details><summary>Code</summary>
+```
++++
+date = "2018-07-22"
+fragment = "404"
+weight = 150
+
+#title = "" # default i18n "404.title"
+#subtitle = "" # default i18n "404.subtitle"
+#redirect_text = "" # default i18n "404.direction"
+#button_text = "" # default i18n "404.button"
+#redirect_url = "" # default /
```

Comment:
Just notice. It's basically a button so we could use the general button declaration instead. Would that make it more consistent?

Just a thought nothing to change in the PR for now.